go.mod

```diff
@@ -1,38 +1,36 @@
 module github.com/osmosis-labs/osmosis/v21
 
-go 1.20
+go 1.21
 
 require (
 	cosmossdk.io/api v0.3.1
 	cosmossdk.io/errors v1.0.0
 	cosmossdk.io/math v1.1.3-rc.1
 	cosmossdk.io/tools/rosetta v0.2.1
 	github.com/CosmWasm/wasmd v0.45.1-0.20231128163306-4b9b61faeaa3
-	github.com/cometbft/cometbft v0.37.2
+	github.com/cometbft/cometbft v0.38.0
```

Comment:
What is forcing this to bump? Both this and sdk import should not be forced to bump when running tidy

Reply:
They are bumped on tidy. Replaces are what matters anyway